user/languages/haskell.md

```diff
@@ -71,3 +71,22 @@ cabal install --only-dependencies --enable-tests
 
 For Haskell projects, `env` and `ghc` can be given as arrays
 to construct a build matrix.
+
+### Multiple Packages in Subdirectories
+
+If you have multiple packages in subdirectories (each containing a `.cabal` file,
+you can enter those directories by setting an environment variable:
```

Comment:
Thanks for this. Do you think it would be more clear to say "specify those directories by setting an environment variable" even if the build command is `cd` ?

Reply:
I agree, your suggestion is more accurate!